packages/api/src/router/app.ts

```diff
@@ -98,6 +99,24 @@ export const appRouter = createTRPCRouter({
         href: input.href,
       });
     }),
+  createMany: permissionRequiredProcedure
+    .requiresPermission("app-create")
+    .input(validation.app.createMany)
+    .output(z.void())
+    .mutation(async ({ ctx, input }) => {
+      await ctx.db.insert(apps).values(
```

Comment:
Add check if `input.length >= 1`. If not just return

Reply:
Not necessary, see the validation schema

Reply:
Not necessary, see the validation schema